src/OpenTelemetry.OpAmp.Client/Internal/Transport/WebSocket/WsTransmitter.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET
+
+using System.Net.WebSockets;
+using Google.Protobuf;
+using OpenTelemetry.Internal;
+using OpenTelemetry.OpAmp.Client.Internal.Utils;
+
+namespace OpenTelemetry.OpAmp.Client.Transport.WebSocket;
+
+internal class WsTransmitter
+{
+    private const int BufferSize = 4096;
```

Comment:
Need to choose realistic buffer size, anyone suggests what are the usuals?

Reply:
May need to tune later then. Prob need to get full session up and have some stats based on different message sizes.
My theory is that the most common message size should be default. 4kB for small messages, maybe increase a new block size to 8kB if there are already 2-3 4kB blocks requested 🤔